greatnovels.sql

```diff
@@ -0,0 +1,152 @@
+DROP DATABASE greatnovels;
```

Comment:
You can delete this file. We only used .sql files as a stand in until we learned migrations, in the real world you would never have .sql files you'd use migrations from the beginning.